apps/site/pages/en/blog/release/v16.14.0.md

```diff
@@ -15,7 +15,7 @@ This release adds experimental support for the import assertions stage 3 proposa
 To keep Node.js ESM implementation as compatible as possible with the HTML spec, import assertions are now required to import JSON modules (still behind the `--experimental-json-modules` CLI flag):
 
 ```mjs
-import info from `./package.json` assert { type: `json` };
+import info from `./package.json` with { type: `json` };
```

Comment:
Claudio don't forget to remove that maybe from codemode

Reply:
It's a blog post about v16

Reply:
I stand corrected :D

Reply:
🤦

Reply:
_Im feeling stupid now_